server/pages/index.js

```diff
@@ -520,9 +520,7 @@ module.exports = function() {
 		} );
 
```

Comment:
Could you also include a recent fix for dev environment here - https://github.com/Automattic/wp-calypso/pull/22508?